.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
+  - "/bin/docker-image-tool.sh"
+  - "/bin/find-spark-home*"
+  - "gradle/"
+  - "**.gradle"
+BASH:
+  - "dev/check-license"
+  - "**.sh"
+DOCS:
+  - "site/"
+  - "!/site/mkdocs.yml"
+  - "**CHANGELOG.md"
+  - "**LICENSE"
+  - "**NOTICE"
+  - "**README.md"
+EXAMPLES:
+  - "examples/"
+COMMON:
+  - "common/"
+API:
+  - "api/"
+  - "python/iceberg/api/"
+  - "python/tests/api/"
```

Comment:
How about separating Python out into its own tag? I don't think that combining it with the others will be helpful for people looking to find PRs.

Reply:
There is currently a python tag. This would tag them as `python` and `api`. I can remove them in favor of keeping it simple and using just `python` if preferred.

Reply:
Yeah, let's keep it simple for now. Thanks!

Reply:
Oh and my mistake. I had originally had a python label and somehow I dropped it in the mix. I agree, simpler is better for now.